flink-end-to-end-tests/test-scripts/docker-mesos-cluster/docker-compose.yml

```diff
@@ -0,0 +1,68 @@
+################################################################################
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+version: '3.5'
+
+networks:
```

Comment:
You can use a custom network or list the keys in `expose`. By default, any service can reach any other service at that service's name [1]. 

> expose
> 
> Expose ports without publishing them to the host machine - they’ll only be accessible to linked services. Only > the internal port can be specified.

https://docs.docker.com/compose/networking/